### PR TITLE
Add handwired LeChiffre Pi

### PR DESCRIPTION
**Pull request by jkaunert**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* Hardware Supported: Handwired, rp_2040W(support for LED, BLE in future revisions), Gateron hot-swap(because i had them in a drawer) or ![HSSv4.4 3D printed sockets](https://github.com/stingray127/handwirehotswap) which is preferred.
+* Hardware Availability: [Build log coming soon](https://philoandgunge.com/handwired/builds/nyaaaah54_rp2040)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/philoandgunge/nyaaaah54_rp2040:default
+
+Flashing example for this keyboard:
+
+    make handwired/philoandgunge/nyaaaah54_rp2040:default:flash
```
Comment (fauxpark) [inline keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/readme.md:17]:
```suggestion
    make handwired/philo_and_gunge/nyaaaah54_rp2040:default

Flashing example for this keyboard:

    make handwired/philo_and_gunge/nyaaaah54_rp2040:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/rules.mk
Comment (fauxpark) [inline keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/rules.mk:1]: This file can be deleted.

File: keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/info.json
Comment (fauxpark) [inline keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/info.json:1]: This file should be renamed to `keyboard.json`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/rules.mk
Comment (jkaunert) [reply keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/rules.mk:1]: fixed

File: keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/info.json
Comment (jkaunert) [reply keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/info.json:1]: fixed

File: keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/keyboard.json
Comment (fauxpark) [inline keyboards/handwired/philo_and_gunge/nyaaaah54_rp2040/keyboard.json:1]: Layout is incorrect.
image

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/rules.mk
Comment (zvecr) [inline keyboards/handwired/are_you_joshing_me/le_chiffre_pi/rules.mk:1]: This file needs to be removed.

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/config.h
Comment (zvecr) [inline keyboards/handwired/are_you_joshing_me/le_chiffre_pi/config.h:1]: This file needs to be removed.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Le Chiffre Pi
+
+![](https://github.com/user-attachments/assets/731698ac-7ee9-4b36-b083-c3377b2a5480)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/handwired/are_you_joshing_me/le_chiffre_pi/readme.md:5]: This needs to be filled in

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
```
Comment (zvecr) [inline keyboards/handwired/are_you_joshing_me/le_chiffre_pi/keymaps/default/keymap.c:2]:
```suggestion
// Copyright 2024 jkaunert (@jkaunert)
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/readme.md (hunk 2 above)
Comment (jkaunert) [reply keyboards/handwired/are_you_joshing_me/le_chiffre_pi/readme.md]: corrected

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+* Keyboard Maintainer: [jkaunert](https://github.com/jkaunert)
+* Hardware Supported: RP2040
+* Hardware Availability: <https://areyoujoshing.me/keyboards>
```
[the remark this answers is not in the document]
Comment (jkaunert) [reply keyboards/handwired/are_you_joshing_me/le_chiffre_pi/readme.md]: fixed

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/keymaps/default/keymap.c (hunk 3 above)
Comment (jkaunert) [reply keyboards/handwired/are_you_joshing_me/le_chiffre_pi/keymaps/default/keymap.c]: fixed

File: keyboards/handwired/are_you_joshing_me/le_chiffre_pi/rules.mk
Comment (jkaunert) [reply keyboards/handwired/are_you_joshing_me/le_chiffre_pi/rules.mk:1]: fixed

**Review by drashna — APPROVED** [review]

**Pull request closed without merging**